src/webui/frontend/src/routes/Dashboard/Dependencies.tsx

```diff
@@ -0,0 +1,75 @@
+import {Button, Card} from 'antd';
+import {LoadingOutlined} from '@ant-design/icons';
+import React from 'react';
+import {connect} from 'react-redux';
+import {bindActionCreators, Dispatch} from 'redux';
+import {newRule, renameRule} from '../../actions/rules';
+import {RuleDashboard} from '../../components/Dashboard';
+import '../../index.css';
+import {getRules} from '../../reducers/rules';
+import * as stateTypes from '../../reducers/types';
+import {Query, Suppression} from '../../store/rules';
+import BasicLayout from '../../layouts/BasicLayout';
+import {history, navigate} from '../../store/history';
+
+import './Alerts.css';
+
+interface OwnProps {
+  path: string;
+  selected?: string;
+}
+
+interface StateProps {
+  rules: stateTypes.SnowAlertRulesState;
+}
+
+interface DispatchProps {
+  newRule: typeof newRule;
+  renameRule: typeof renameRule;
+}
+
+type Props = OwnProps & StateProps & DispatchProps;
+
+class Dependencies extends React.PureComponent<Props> {
+      
+
+
+  render() {
+    
+    return (
+      <BasicLayout>
+        <Card
+          className={'card'}
+        
+        >
+            <div>
+              <LoadingOutlined /> Loading Rules...
+            </div>
+          ) : (
```

Comment:
this seems malformed -- did you mean to include it in a separate branch, maybe?

Reply:
I had started working on a barebones data dependencies page, since this was a draft I didn't completely remove it yet, for working on it later. Will remove it now :)